contrib/gitian-keys/keys.txt

```diff
@@ -30,3 +30,4 @@ ED9BDF7AD6A55E232E84524257FF9BDBCC301009 Sjors Provoost
 AEC1884398647C47413C1C3FB1179EB7347DC10D Warren Togami
 79D00BAC68B56D422F945A8F8E3A8F3247DBCBBF Willy Ko
 71A3B16735405025D447E8F274810B012346C9A6 Wladimir J. van der Laan
+9D3CC86A72F8494342EA5FD10A41BDC3F4FAFF1C Aaron Clauson (sipsorcery)
```

Comment:
It should be in the first name alphabetic order.

Reply:
Ahh somehow didn't spot that. Fixed now. Woo hoo straight to the top of the list (if accepted that is)!